content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -54,6 +54,7 @@ different Kubernetes components.
 | `DevicePlugins` | `true` | Beta | 1.10 | |
 | `DynamicKubeletConfig` | `false` | Alpha | 1.4 | 1.10 |
 | `DynamicKubeletConfig` | `true` | Beta | 1.11 | |
+| `DynamicProvisioningScheduling` | `false` | Alpha | 1.11 | 1.11 |
```

Comment:
Is the feature introduced in 1.11 and it is removed in 1.12?

Reply:
add in v1.11, but seems not finish completed, https://github.com/kubernetes/kubernetes/pull/63232/commits/95b530366aeea7c379c50424f58bcd2f27bed921

Reply:
Then you should remove `1.11` from the last column